package.json

```
@@ -32,6 +32,7 @@
     "chalk": "^2.3.1",
     "node-fetch": "^2.0.0",
     "keytar": "^4.1.0",
-    "inquirer": "^5.1.0"
+    "inquirer": "^5.1.0",
+    "https": "^1.0.0"
```

since we are in nodejs environment, we don't need this dependency

```suggestion
    "inquirer": "^5.1.0"
```